### PR TITLE
put failed events into a separate tree

### PR DESCRIPTION
**Pull request by hroskes**
also some fixes in probability names for WH

and merge

### CODE REVIEW

File: AnalysisStep/test/prod/samples_2016_MC.csv [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+#ggTo2e2tau_0PMH125_MCFM701,,  -1,   ,,/GluGluToHiggs0PMToZZTo2e2tau_M125_GaSM_13TeV_MCFM701_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM,dbs,,2,PROCESS_CR=False;APPLY_K_NNLOQCD_ZZGG=1;ADDLOOSEELE=True,RecoProbabilities.py;LHEProbabilities_GG_SIG_0PM_H125_MCFM.py,#FIXME: x-section??
+#ggTo2mu2tau_0PMH125_MCFM701,, -1,   ,,/GluGluToHiggs0PMToZZTo2mu2tau_M125_GaSM_13TeV_MCFM701_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM,dbs,,4,PROCESS_CR=False;APPLY_K_NNLOQCD_ZZGG=1;ADDLOOSEELE=True,RecoProbabilities.py;LHEProbabilities_GG_SIG_0PM_H125_MCFM.py,#FIXME: x-section??
+
+#ggTo4e_0PMH125Contin_MCFM701,,      -1,   ,,/GluGluToHiggs0PMContinToZZTo4e_M125_GaSM_13TeV_MCFM701_pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM,dbs,,1,PROCESS_CR=False;APPLY_K_NNLOQCD_ZZGG=1;ADDLOOSEELE=True,RecoProbabilities.py;LHEProbabilities_GG_BSI_0PM_H125_MCFM.py,#FIXME: x-section??
```
Comment (hroskes) [inline AnalysisStep/test/prod/samples_2016_MC.csv]: Just to make sure: LHEProbabilities_GG_BSI is correct?  Not BSI10?

**Comment by hroskes** [comment]
Ok, done, I was just being consistent with what you had put

**Comment by usarica** [comment]
I think it would be more consistent to name them ghw or ghz, for example:
https://github.com/usarica/ZZAnalysis/blob/miniAOD_80X_newReWgt/AnalysisStep/test/prod/pyFragments/LHEProbabilities_ZH_SIG_0PM_H130_POWHEG.py#L147

ghv should only be there if ghw_i=ghz_i is assumed. Note that we will need to make such separation for going from (B+W3, W1+W2) basis to (B,W_i) basis in couplings.

**Comment by hroskes** [comment]
Yes, I understand.  It's switched now.

**Comment by usarica** [comment]
Do you mind also changing the names in the synchronizers and sync scripts? (anything that uses Had_WH for categorization etc.)

**Comment by hroskes** [comment]
Not sure what you mean but I can't look tonight.  Feel free to merge and then change it.

**Comment by hroskes** [comment]
The probabilities are still called HadWH_SIG_ghw1_1_JHUGen_JECNominal.  It's just the Production id that is Had_WH consistent with MELAOptionParser.cc and TVar.hh.

**Pull request merged**